src/compass/s1_static_layers.py

```diff
@@ -0,0 +1,86 @@
+import journal
+import re
+import time
+
+from datetime import timedelta
+from compass.utils.geo_runconfig import GeoRunConfig
+from compass.utils.yaml_argparse import YamlArgparse
+from compass.utils.helpers import bursts_grouping_generator, get_module_name
+from compass import s1_rdr2geo
+from compass import s1_geocode_metadata
+
+
+def _make_rdr2geo_cfg(yaml_runconfig_str):
+    '''
+    Make a rdr2geo specific runconfig with latitude, longitude, and height
+    layers enabled for static layer product generation while preserving all
+    other rdr2geo config settings
+
+    Parameters
+    ----------
+    yaml_runconfig_str: str
+        Workflow runconfig as a string
+
+    Returns
+    -------
+    rdr2geo_cfg: dict
+        Dictionary with rdr2geo longitude, latitude, and height layers
+        enabled. All other rdr2geo parameters are from *yaml_runconfig_str*
+    '''
+    # If any of the requisite layers are false, make them true in yaml cfg str
+    for layer in ['latitude', 'longitude', 'incidence_angle']:
+        re.sub(f'compute_{layer}:\s+[Ff]alse', f'compute_{layer}: true',
+               yaml_runconfig_str)
+
+    # Load a GeoRunConfig from modified yaml cfg string
+    rdr2geo_cfg = GeoRunConfig.load_from_yaml(yaml_runconfig_str,
+                                              workflow_name='s1_cslc_geo')
+
+    return rdr2geo_cfg
+
+
+def run(cfg: GeoRunConfig):
+    '''
+    Run static layers workflow (i.e., generate static layers,
+    geocode them, create product HDF5) with user-defined
+    args stored in dictionary runconfig *cfg*
+
+    Parameters
+    ---------
+    cfg: GeoRunConfig
+        GeoRunConfig object with user runconfig options
+    '''
+
+    module_name = get_module_name(__file__)
+    info_channel = journal.info(f"{module_name}.run")
+    info_channel.log(f"Starting {module_name} burst")
+
+    # Start tracking processing time
+    t_start = time.time()
```

Comment:
```suggestion
    t_start = time.perf_counter()
```
For whatever reason the change above also needs to be done [here](https://github.com/opera-adt/COMPASS/blob/de33cfad9692efc08e9e53296a2e6d1fdad410df/src/compass/s1_geocode_metadata.py#L84) to address [this](https://github.com/opera-adt/COMPASS/pull/177#issuecomment-1638703373).